private/buf/cmd/buf/testdata/convert/bin_json/convert_test.go

```diff
@@ -0,0 +1,200 @@
+package bin_json
+
+import (
+	"github.com/bufbuild/buf/private/buf/cmd/buf"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd"
+	"github.com/bufbuild/buf/private/pkg/app/appcmd/appcmdtesting"
+	"strings"
+	"testing"
+)
+
+// Unfortunately os.Chdir is very flaky in tests and cannot be done for only one go routine
+// Putting a test file here was the best option I could come up with.
+func TestConvertDir(t *testing.T) {
+	cmd := func(use string) *appcmd.Command { return buf.NewRootCommand("buf") }
+	t.Run("default-input", func(t *testing.T) {
+		appcmdtesting.RunCommandExitCodeStdout(
```

Comment:
You can do i.e. `buf convert testdata/convert/bin_json ...` correct? If so, if you move this test up a few levels, convert to this. If not, talk to me about why so (offline, moving off this PR after this round of comments)